src/main/java/io/strimzi/kafka/bridge/http/HttpLogging.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2019, Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+
+package io.strimzi.kafka.bridge.http;
+
+import io.vertx.core.http.HttpServerRequest;
+import io.vertx.core.http.HttpServerResponse;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Utility class for logging HTTP requests/responses
+ */
+public class HttpLogging {
+
+    private static final Logger log = LoggerFactory.getLogger(HttpLogging.class);
+
+    public static void logRequest(HttpServerRequest request) {
+        log.info("Request: method = {}, path = {}", request.method(), request.path());
+        log.debug("Request: headers = {}", request.headers());
```

Comment:
When the bridge is serving requests concurrently how will you correlate the request and response log messages? Is there some correlation id which the request and response share, perhaps?